shimmingtoolbox/load_nifti.py

```diff
@@ -0,0 +1,165 @@
+#!usr/bin/env python3
+# -*- coding: utf-8
+
+import os
+import logging
+import numpy as np
+import nibabel as nib
+import json
+import math
+
+logger = logging.getLogger(__name__)
+PHASE_SCALING_SIEMENS = 4096
+
+
+def load_nifti(path_data):
+    """
+    Load data from a NIFTI type file with dcm2bids.
+    Args:
+        path_data (str): absolute or relative path to the directory the acquisition data
+    Returns:
+        info (Nifti1Image.Header): List containing all information from every Nifti image
+        json_info (dict): List containing all information in JSON format from every Nifti image
+        niftis (ndarray): 5D array of all acquisition in time (x, y, z, echo, volume)
+
+    Note:
+        If 'path' is a folder containing niftis, directly output niftis. It 'path' is a folder containing acquisitions,
+        ask the user for which acquisition to use.
+    """
+    if not os.path.exists(path_data):
+        raise RuntimeError("Not an existing NIFTI path")
+
+    file_list = []
+    [file_list.append(os.path.join(path_data, f)) for f in os.listdir(path_data) if f not in file_list]
+
+    nifti_path = ""
+    if all([os.path.isdir(f) for f in file_list]):
+        acquisitions = [f for f in file_list if os.path.isdir(f)]
+        logging.info("Multiple acquisition directories in path. Choosing only one.")
+    elif all([os.path.isfile(f) for f in file_list]):
+        logging.info("Acqusition directory given. Using acquisitions.")
+        nifti_path = path_data
+    else:
+        raise RuntimeError("Directories and files in input path")
+
+    if not nifti_path:
+        for i in range(len(acquisitions)):
+            logging.info("{}:{}\n".format(i, os.path.basename(file_list[i])))
+            print("{}:{}\n".format(i, os.path.basename(file_list[i])))
```

Comment:
Copied from: https://github.com/shimming-toolbox/shimming-toolbox-py/pull/46#discussion_r465966701

Since our `setup.py` explicitly requires Python 3.6 or greater, we have access to f-strings for string formatting, which is a bit tidier for future readers.

See these resources for details:
- https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python
- https://docs.python.org/3.6/reference/lexical_analysis.html#f-strings